R/install-svn.r

```diff
@@ -64,6 +70,16 @@ remote_download.svn_remote <- function(x, quiet = FALSE) {
     stop("There seems to be a problem retrieving this SVN-URL.", call. = FALSE)
   }
 
+  if (!is.null(x$revision)) {
+    pwd <- getwd()
```

Comment:
FWIW, a simpler pattern is `pwd <- setwd(bundle)` - but I'll make this change locally.